Sources/StreamChat/Audio/AudioPlayer/AudioPlaying.swift

```diff
@@ -289,8 +272,6 @@ open class StreamRemoteAudioPlayer: AudioPlaying, AppStateObserverDelegate {
 
                 value.isSeeking = false
 
-                value.state = player.rate != 0 ? .playing : .paused
```

Comment:
With the `timeControlStatus` observer update, this will be handled by the `addTimeControlStatusObserver` that is being registered below.